packages/formdata-event/ts_src/environment/event.ts

```diff
@@ -9,7 +9,7 @@
  * additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
-export const constructor = window.Event;
+export const constructor: typeof Event = window.Event;
```

Comment:
This probably needs a comment. Without this explicit type, TS seems to believe that `window.Event` is the wrapper from `wrappers/event.ts` at this point in time, but that wrapper is defined in terms of this variable, which causes a cycle in the type.

Reply:
Added.